css/mobile-01web.css

```diff
@@ -87,7 +87,7 @@ body {
 	list-style:none;
 }
  .arrowList li {
-	background: url(../images/arrow.png) no-repeat 0 2px;
+	background: url(img/arrow.png) no-repeat 0 2px;
```

Comment:
Same in this file, please revert the changes or move the images back into css/img